content/docs/2.8/concepts/scaling-deployments.md

```diff
@@ -226,6 +226,22 @@ metadata:
 
 The presensce of this annotation will pause autoscaling no matter what number of replicas is provided. The above annotation will scale your current workload to 0 replicas and pause autoscaling. You can set the value of replicas for an object to be paused at to any arbitary number. To enable autoscaling again, simply remove the annotation from the `ScaledObject`definition.
 
+### Activating and Scaling thresholds
```

Comment:
Would you mind updating all versions please?

Reply:
No no, this change applies only after next version because requires changes on every scaler